pages/about/groups/task-forces/matf.md

```diff
@@ -32,65 +32,71 @@ Find out about the activities of the Mobile Accessibility (Mobile A11y) Task For
 
 ## Objectives
 
-The objectives of the Mobile A11y Task Force are to produce resources for how to apply the Web Content Accessibility Guidelines (WCAG) to mobile — including, but not limited to, native mobile apps; mobile web apps; mobile web content; and hybrid apps that use web components inside native mobile apps — and to make sure mobile apps are considered in new accessibility guidelines.
+The Mobile Accessibility Task Force:
+
+- produces resources for how to apply the Web Content Accessibility Guidelines (WCAG) to mobile, and 
+- makes sure mobile apps are considered in new accessibility guidelines.
+
+Mobile includes, but is not limited to, native mobile apps, mobile web apps, and hybrid apps that use web components inside native mobile apps.
 
 ## Work statement
 
-The Mobile A11y Task Force has regrouped in 2024 and a new Work Statement will be published soon.
+To learn about the focus, scope and deliverables of the task force, see the [Mobile Accessibility Task Force Work Statement](/about/groups/task-forces/matf/work-statement/).
 
 ## Current work
 
-The Mobile A11y Task Force supports the deliverables of the [Accessibility Guidelines (AG) Working Group](/about/groups/agwg/).
+The Mobile Accessibility Task Force supports the deliverables of the [Accessibility Guidelines (AG) Working Group](/about/groups/agwg/).
+
+For details of the current work, see the [Mobile Accessibility Task Force GitHub repository](https://github.com/w3c/matf/issues).
 
-For details of the current work, see the [Mobile A11y Task Force GitHub repository](https://github.com/w3c/matf/issues).
+For more information, see [Mobile Accessibility at W3C](https://www.w3.org/WAI/standards-guidelines/mobile/).
 
 ## Contribute to the work
 
-The Mobile A11y Task Force welcomes input from the global accessibility community on mobile accessibility work.
+The Mobile Accessibility Task Force welcomes input from the global accessibility community on mobile accessibility work.
 
 ### Contribute without joining the task force
 
-You can contribute to the work without being a member of the task force in the [Mobile A11y GitHub repository](https://github.com/w3c/Mobile-A11y-TF-Note/issues):
+You can contribute to the work without being a member of the task force in the [Mobile Accessibility GitHub repository](https://github.com/w3c/matf/issues):
```

Comment:
@tamsinewing555 This line was previously updated by @iadawn, see https://github.com/w3c/wai-website/commit/b6e4b134d699db928ca708ca3b2e431794864b37